lib/rubocop-rspec.rb

```diff
@@ -37,16 +37,26 @@
 # We have to register our autocorrect incompatibilities in RuboCop's cops
 # as well so we do not hit infinite loops
 
+# rubocop:disable Style/Documentation
 module RuboCop
   module Cop
     module Layout
-      class ExtraSpacing # rubocop:disable Style/Documentation
+      class ExtraSpacing
         def self.autocorrect_incompatible_with
           [RSpec::AlignLeftLetBrace, RSpec::AlignRightLetBrace]
         end
       end
     end
+
+    module Style
+      class TrailingCommaInArguments
+        def self.autocorrect_incompatible_with
+          [RSpec::Capybara::CurrentPathExpectation]
```

Comment:
doesn't we override whatever is set in TrailingCommaInArguments, potentially breaking other extensions or future cops? Shouldn't we declare the incompatibility only in our own cop? If the incompatibility has to be declared in both cops, should we include a module which would then append to whatever `super` returns?

cc: @bquorning

Reply:
You’re right.

Reply:
You are right.
I tried declaring incompatibility only in our own cop, but it did not work out as expected.

So I'm trying to solve this problem in the following way:
> If the incompatibility has to be declared in both cops, should we include a module which would then append to whatever super returns?

Result of running RSpec when we declared incompatibility only in our own cop:
```
bundle exec rspec spec/rubocop/cli/autocorrect_spec.rb:35
Run options: include {:focus=>true, :locations=>{"./spec/rubocop/cli/autocorrect_spec.rb"=>[35]}}

Randomized with seed 40281

RuboCop::CLI --autocorrect
  when corrects `RSpec/Capybara/CurrentPathExpectation` with `Style/TrailingCommaInArguments`
    autocorrects be compatible with each other (FAILED - 1)

Failures:

  1) RuboCop::CLI --autocorrect when corrects `RSpec/Capybara/CurrentPathExpectation` with `Style/TrailingCommaInArguments` autocorrects be compatible with each other
     Failure/Error:
       expect(File.read('spec/example.rb')).to eq(<<-RUBY)
         expect(page).to have_current_path(
           some_path(
             id: id,
           ), ignore_query: true
         )
       RUBY
     
       expected: "        expect(page).to have_current_path(\n          some_path(\n            id: id,\n          ), ignore_query: true\n        )\n"
            got: "        expect(page).to have_current_path(\n          some_path(\n            id: id,\n          ),, ignore_query: true\n        )\n"
     
       (compared using ==)
     
       Diff:
       @@ -1,6 +1,6 @@
                expect(page).to have_current_path(
                  some_path(
                    id: id,
       -          ), ignore_query: true
       +          ),, ignore_query: true
                )
       
     # ./spec/rubocop/cli/autocorrect_spec.rb:40:in `block (3 levels) in <top (required)>'
     # ./spec/support/cli_spec_behavior.rb:21:in `block (2 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.6.0/gems/rubocop-1.25.1/lib/rubocop/rspec/shared_contexts.rb:30:in `block (4 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.6.0/gems/rubocop-1.25.1/lib/rubocop/rspec/shared_contexts.rb:30:in `chdir'
     # ./vendor/bundle/ruby/2.6.0/gems/rubocop-1.25.1/lib/rubocop/rspec/shared_contexts.rb:30:in `block (3 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.6.0/gems/rubocop-1.25.1/lib/rubocop/rspec/shared_contexts.rb:7:in `block (2 levels) in <top (required)>'

Finished in 0.13613 seconds (files took 1.13 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/rubocop/cli/autocorrect_spec.rb:35 # RuboCop::CLI --autocorrect when corrects `RSpec/Capybara/CurrentPathExpectation` with `Style/TrailingCommaInArguments` autocorrects be compatible with each other

Randomized with seed 40281
```